content/articles/build-a-contact-form-with-php/index.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: engineering-education
+status: draft
+published: false
+url: /build-a-contact-form-with-php/
+title: Build a contact form with PHP
+description: Learn how to add a contact form to your PHP website.
+author: samuel-torimiro
+date:
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/build-a-contact-form-with-php/hero.jpg
+    alt: PHP example image
+---
+
+[PHP](https://www.php.net/) is a programming language used for creating interactive web applications. It's a widely used language that runs on the server.
+<!--more-->
+In this tutorial, will build a contact form with PHP. It will include the following features, form validation, the ability to send emails using Google Gmail SMTP server, and [PHPMailer library](https://github.com/PHPMailer/PHPMailer).
+
```

Comment:
```suggestion
In this tutorial, will build a contact form with PHP. It will include the following features, form validation, the ability to send emails using the Google Gmail SMTP server, and the [PHPMailer library](https://github.com/PHPMailer/PHPMailer).

```